packages/core/src/components/SlideTransition/SlideTransition.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { forwardRef } from "react";
+import { motion } from "framer-motion";
+import cx from "classnames";
+import { slideAnimationVariants, slideAnimationTransition } from "./utils/animationVariants";
+import { SlideTransitionProps } from "./SlideTransition.types";
+import styles from "./SlideTransition.module.scss";
+
+const SlideTransition = forwardRef(
+  ({ direction, children, className }: SlideTransitionProps, ref: React.ForwardedRef<HTMLDivElement>) => {
+    return (
+      <motion.div
+        ref={ref}
+        custom={direction}
+        variants={slideAnimationVariants}
+        initial="initial"
+        animate="enter"
+        exit="exit"
+        transition={slideAnimationTransition}
+        className={cx(styles.slide, className)}
```

Comment:
Does it really need a className?

Reply:
for now it would be an internal component, but in case we decide to expose a few animation components next that is an hachana lemazgan, I gave it all the vibe props 
`export interface SlideTransitionProps extends VibeComponentProps {`

Reply:
Ok. Anyway notice you are not exporting the other component. Not sure if it's intended or not

Reply:
nope, will do

Reply:
Done